doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -16,18 +16,18 @@
 <a name="1.1"></a>
 ## 1.1 Introduction
 
-[Kubernetes](https://kubernetes.io/) is a "portable, extensible, open-source platform for managing containerized workloads and services, that facilitates both declarative configuration and automation. It has a large, rapidly growing ecosystem. Kubernetes services, support, and tools are widely available." [[kubernetes.io](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/)]. Kubernetes is developed as an open source project in the [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) repository of GitHub.
+The intention of this Reference Architecture is to develop a usable container based platform for the Telecom operator community.  The RA will be based on the standard Kubernates platform where ever possible. This Reference Architecture for Kubernetes will describe the high level system components and their interactions, taking the goals and requirements of the [Reference Model](../../../ref_model/chapters/chapter01.md) and mapping them to real-world Kubernetes (and related) components. This document needs to be sufficiently detailed and robust such that it can be used to guide the production deployment of Kubernetes within an operator, whilst being flexible enough to evolve with and remain aligned with the wider Kubernetes ecosystem outside of Telco.
```

Comment:
> The intention of this Reference Architecture is to develop a usable container based platform for the Telecom operator community.  The RA will be based on the standard Kubernates platform where ever possible.

Couple of requests please:
1. Inconsistent spacing between sentences
1. `Kubernates` -> `Kubernetes`
1. Can we modify the wording please - see below for my suggestion. The scope has been discussed a number of times and is not restricted to containers.

> The intention of this Reference Architecture is to develop a usable Kubernetes based platform for the Telecom operator community.